### PR TITLE
Revert "Magazine page ready"

### PR DESCRIPTION
**Pull request by tigamadou**
Reverts WCanirinka/smashingmagazine-page#1

### CODE REVIEW

**Comment by WCanirinka** [comment]
Requesting for a review. We saw a thumb up and thought it was a go ahead and merged. Kindly review our work

**Review by Redvanisation — APPROVED** [review]
# Project approved ✔️ 

### Hello team 🌟 

**_Nice work on this project, it seems that you guys have met the requirements so I am approving your project. 👍_** 

## However, for the next project please do not merge and switch branches without approval, all changes should be done from the same pull request.

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Comment by tigamadou** [comment]
Thank you.

**Pull request merged**
